playbooks/byo/preflight/action_plugins/verify_openshift_image.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+
+from ansible.plugins.action import ActionBase
+
+
+class ActionModule(ActionBase):
+
+    def run(self, tmp=None, task_vars=None):
+        ''' handler verifying openshift images are available '''
+        if task_vars is None:
+            task_vars = dict()
+
+        result = super(ActionModule, self).run(tmp, task_vars)
+
+        component  = self._task.args.get('component', None)
```

Comment:
`None` is the default "default value".

There is a double space before the equal sign that will fail the linting tools :)

```python
component = self._task.args.get('component')
```